templates/_helpers.yaml

```diff
@@ -53,15 +53,21 @@
 
 {{/* User defined Airflow environment variables */}}
 {{- define "custom_airflow_environment" }}
```

Comment:
@kushalmalani since this has taken on some new meaning what if we named this helper just `airflow_environment` or something, since it now contains custom envs, as well as the standard envs.